tests/pos/i4582.scala

```diff
@@ -0,0 +1,2 @@
+trait T[A: Numeric]
+class TX[A: Numeric] extends T[A]
```

Comment:
This doesn't exercice the code you changed but I would add a test with an argument list. E.g.
```scala
trait T2[A: Numeric](x: Int)
class T2X[A: Numeric] extends T[A](1)
```

Reply:
I added it.